src/keyboard-layout-manager-linux.cc

```diff
@@ -1,16 +1,19 @@
 #include "keyboard-layout-manager.h"
 
-using namespace v8;
+#include <X11/XKBlib.h>
+#include <X11/Xutil.h>
+#include <X11/extensions/XKBrules.h>
```

Comment:
This header is part of an extension, which means that not everyone has it, this probably should have been a breaking change in the package.json